.travis.yml

```diff
@@ -46,68 +82,6 @@ jobs:
       install:
       script:
         - docker build -f Dockerfile -t openwpm .
-        - ./.deploy-to-dockerhub.sh
-    - language: node_js
```

Comment:
Please restore the tests for webext-instrumentation. If you find a cleaner way, it'd be great, otherwise just restore as they were. Thanks :)